versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/common-issues.md

```diff
@@ -0,0 +1,6 @@
+---
+title: AWS Marketplace Common Issues
```

Comment:
We should probably specify Rancher Prime, for Algolia search and SEO reasons

Reply:
Updated title for AWS/Azure common issues per 0e8b62a573ba335cf2b6950ebdc88c2b4645f6d1